brozzler/worker.py

```diff
@@ -260,6 +259,38 @@ def thumb_jpeg(self, full_jpeg):
         img.save(out, "jpeg", quality=95)
         return out.getbuffer()
 
+    def should_ytdlp(self, logger, site, page, page_status, skip_av_seeds):
```

Comment:
Just had a thought, is it also worth adding an attempt to import ydl and erroring out if it can't? eg, we shouldn't ytdlp if the optional extra is missing.

```python
try:
    import yt_dlp
except ImportError:
    return False
```

Reply:
Oh good idea, probably with log message

Reply:
Done in a0a06b5d3316557830dcbc480da47d5ac61ad17a in the other PR.